reporting-engine-worker/src/main/java/org/ihtsdo/termserver/scripting/reports/release/PackageComparisonReport.java

```diff
@@ -119,7 +124,7 @@ public Job getJob() {
 	}
 
 	@Override
-	protected void loadProjectSnapshot(boolean fsnOnly) throws TermServerScriptException, InterruptedException, IOException {
+	protected void loadProjectSnapshot(boolean fsnOnly) {
 	}
```

Comment:
I'd put a comment in here to explain why it's necessary to have an empty method.  In this case you'll explain that this process does not need to hold a snapshot in memory, so you're overriding the default behaviour in doing nothing here.